core/src/main/java/com/sequenceiq/cloudbreak/service/securitygroup/DefaultSecurityGroupCreator.java

```diff
@@ -47,7 +47,7 @@
         Set<SecurityGroup> securityGroups = new HashSet<>();
 
         //create default strict security group
-        SecurityGroup onlySshAndSsl = createSecurityGroup(user, "only-ssh-and-ssl", "Only the 22 and 443 ports could be reached from any IP.");
+        SecurityGroup onlySshAndSsl = createSecurityGroup(user, "only-ssh-and-ssl", "Ports 22 (SSH) and 443 (Cloudbreak gateway) are opened from anywhere.");
```

Comment:
Open ports: 22 (SSH) 443 (HTTPS)
Open ports: 8080 (Ambari) 8500 (Consul) 50070 (NN) 8088 (RM WEB) 

This message doesn't make sense